base/tools/src/main/java/com/netscape/cmstools/pkcs12/PKCS12ImportCLI.java

```diff
@@ -127,14 +141,43 @@ public void execute(CommandLine cmd) throws Exception {
             PKCS12Util util = new PKCS12Util();
             util.setTrustFlagsEnabled(trustFlagsEnabled);
 
+            // load PKCS12 object from file
             PKCS12 pkcs12 = util.loadFromFile(filename, password);
 
+            Collection<PKCS12CertInfo> caCerts = new ArrayList<>();
+            Collection<PKCS12CertInfo> userCerts = new ArrayList<>();
+
+            for (PKCS12CertInfo certInfo : pkcs12.getCertInfos()) {
+                byte[] keyID = certInfo.getKeyID();
+                if (keyID == null) {
+                    // no key -> CA cert
+                    caCerts.add(certInfo);
+                } else {
+                    // has key -> user cert
+                    userCerts.add(certInfo);
+                }
```

Comment:
Is defined in the specification that all cert without keys are CA  or is this an assumption done in the command? Maybe in real scenario it does not provide any real difference.